public/css/ui_patterns/article.css

```diff
@@ -24,6 +24,21 @@
 	text-rendering: optimizeSpeed;
 }
 
+/* P A R R A F O */
+.article p {
+	line-height: 1.8;
+	font-size: 16px;
+	margin-top: 0;
+	margin-bottom: 40px;
+	color: #282828;
+	text-align: justify;
+}
+
+/* I N L I N E    L I N K */
+.article p a {
+	color: #282828;
+	border-bottom: 3px solid #c3dddb;
+}
```

Comment:
Falta el efecto de hover: "hover state affects text only, not the underline (text appears in 60% transparency as in all other active elements on the site".

Este efecto esta interesante, investiga como hacer elementos transparentes, en especial el texto.